x-pack/plugins/fleet/common/services/limited_package.ts

```diff
@@ -7,9 +7,10 @@
 
 import type { PackageInfo, AgentPolicy, PackagePolicy } from '../types';
 
-// Assume packages only ever include 1 config template for now
 export const isPackageLimited = (packageInfo: PackageInfo): boolean => {
-  return packageInfo.policy_templates?.[0]?.multiple === false;
+  return !!packageInfo.policy_templates?.find(
```

Comment:
Rather than coerce this to a boolean, it might make more sense to use `.some` to directly return a boolean based on this condition.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some